### DISCUSSION TITLE
现在的版本还卡吗

### DISCUSSION THREAD

**Discussion by NEXWX** [c0re100/qBittorrent-Enhanced-Edition#431] [Q&A]
如题，本来我一直是在用qb的，但是之前理新后打开qb卡得要死，就用回比特慧星了

请问现在好些了吗

**Comment by Retia-Adolf** [comment]
对于我来说所有 4.3.8.10 以后的版本都是卡的（至少图形界面的版本如此），虽然我可能是特例。因为 issue 里有人好像说了，把 高级>网络接口 从任意网络接口 改为固定的网口就好转了。虽然对于我来说，任意网络接口包含手机共享的网络时的确会造成上传下载卡顿，但是对图形界面没有影响，改为特定网口对新版的卡顿也没有好转。

https://github.com/c0re100/qBittorrent-Enhanced-Edition/issues/280
这个 issue 里也有人说“是tracker太多导致的”，但旧版没有问题，更新新版之后反而会卡顿，这个理由是不可接受的。而且无论改了什么设置之后不卡了，看起来都是权宜之计。


https://github.com/qbittorrent/qBittorrent/issues/17680#issuecomment-1279282402
qB 官方在升级到 libtorrent 2 后也有大量反映内存占用过多的 issues，虽然我正在用的 qbEE 4.3.8.10 (libtorrent 1.2.14.0) 在我升级到了 64 GB 内存之后也是有多少占用多少，有时候把 nice 值过高的 firefox 卡到被杀死……
然后这个回复里就有人对比了 ut 和 qB，ut 2.2.1 2500个种子，共 20TB 做种，加载 14 秒，内存占用不过是 300 MB 级别，反映了 qB 的低性能。

比特彗星的长效种子有点私有网络的倾向，本身又是比较重的专有软件，我就懒得用了。但现在有些后悔迁移到 qB。但是现在我有太多重命名和部分勾选的种子在 qbEE 里，而且其它的 BT 软件又没有 qbEE 的反吸血能力，现在我又没法升级，感觉走进死胡同出不来，有点绝望了。

https://github.com/topics/anti-leech
搜索了一下，发现主要有两个给 ut，一个给 transmission 附加反吸血功能的项目。其中一个只提到了反迅雷，其它两个不知道覆盖面跟 qbEE 相比怎么样，也不知道 Auto Ban 的原理有什么不同。

**Reply by SeaHOH** [reply]
我虽然给 uTorrent 写了反吸血小工具，但现在已经没有使用了。
因为受不了uTorrent 3 不断累计的 bug，先是找到这里，尝试使用了几天 qBEE，也受不了。当时正好是 qB 更改网络监听方式的时候，完全不适合我的网络配置，而且运行在旧系统中还特容易崩溃。然后我发现了 Tixati，反吸血跟 qBEE 差不多，主要靠过滤字符串，就是界面和设定与 uTorrent、qB 之流有较大差别，需要时间来习惯，也有一些小毛病 (比如超随便的多语言支持)，但都还能忍受。
